landscape/__init__.py

```diff
@@ -22,7 +22,7 @@
 # 3.3:
 #  * Add new schema for the "registration" message, providing Juju information
 #
-SERVER_API = b"3.3"
+SERVER_API = b"3.4"
```

Comment:
Since adding an optional field to the registration message, doesn't break anything or change the format of the api, it's not necessary to create a new registration message/ new version of the api.

Reply:
OK, will make that change.